src/index.ts

```diff
@@ -0,0 +1,37 @@
+import http = require("http");
+import fs = require("fs");
+
+const postData: http.RequestListener = (
+  request: http.IncomingMessage,
+  response: http.ServerResponse
+) => {
+  if (request.method === "POST") {
```

Comment:
You should think about "error first" approach.

Reply:
Implemented "error first" approach